app/controllers/items/detail.js

```diff
@@ -436,6 +436,27 @@ export default GoodcityController.extend(
         this.onSearchCountry(field, searchText);
       },
 
+      onSaleableChange() {
+        const item = this.get("item");
+        const saleable = item.get("saleable");
+        item.set("saleable", !saleable);
+        this.runTask(async () => {
+          try {
+            await item.save();
+          } catch (e) {
+            item.rollbackAttributes();
+            throw e;
+          }
+        }, ERROR_STRATEGIES.MODAL);
+
+        let loadingView = Ember.getOwner(this)
+          .lookup("component:loading")
+          .append();
+        item.save().finally(() => {
+          loadingView.destroy();
+        });
```

Comment:
Remove this entire line.It's already done above. `runTask` takes care of spinner.

Reply:
done 👍 
i had kept it by mistake. have removed it